lib/events-reader.js

```diff
@@ -189,10 +189,10 @@ module.exports = function (harvesterApp) {
                         });
 
                 } else {
-                    that.reschedule(1000);
+                    that.reschedule(0);
                 }
             } else {
-                that.reschedule(1000);
+                that.reschedule(0);
```

Comment:
when is doc null, don't recall now after all this time
is stream.read returning null without wait if there are no results ?
if that's the case then perhaps this timeout value has to be set to 500/1000, otherwise it will continuously loop without pause